src/api/app/models/flag.rb

```diff
@@ -37,9 +37,9 @@ def validate_duplicates
 
   scope :with_types, ->(type) { where(flag: type) }
   scope :with_repositories, ->(repo_name) { where(repo: repo_name) }
```

Comment:
I know, this was already there, but this scope seems to obfuscate the code more than it would spare use characters, no?

  ``` .with_repositories(repo) ```
vs
  ``` .where(repo: repo.name) ```